_posts/2016-04-19-how-to-contribute-to-gis-utah-gov.md

```diff
@@ -0,0 +1,66 @@
+
+#### Creating New Content
+
+When you, your group, or your employer have content that would benefit the audience of gis.utah.gov, please consider publishing it here. The ease of contributing stems from the website being hosted on [GitHub.com](https://github.com/home). Once you have gone through the process once, you will be an expert.
+
+> GitHub fosters a fast, flexible, and collaborative development process that lets you work on your own or with others.
+
+#### A Post or a Page
+
+The first decision an author needs to make is whether to create a `Post` or a `Page`. Differentiating between the two is done by the type of content being created. If your content is newsy or time sensitive, then it would be considered a `Post`. If the content has permanent value and fits into the organizational structure of the website, then it would considered a `Page`. If you are having trouble deciding, _always_ choose a `Post`.
+
+#### The Starting Point
+
+![templater screenshot]({{ "/images/contributing/content-templater.png" | prepend: site.baseurl }})
+{:.inline-text-right}
+The ever resourceful [Scott Davis](https://twitter.com/@SThomasDavis) created a great tool, the [Templator](http://gis.utah.gov/templater), to create a starting point for authoring new content. This is _the_ entry point for all authors creating new content.
+
+**Start** by creating a title. This will be used to create the url on the website and also the heading for the article. **Next** choose your content type based on the decision above. Fill out the Author information - This will be displayed above the title on the post. Please use your **full name**. Once you **click** `Generate`, the templater will present you with a `File Path` and `Contents`.
+
+#### GitHub
+
+You will need a GitHub account to participate, so [create a free GitHub account](https://github.com/join) if you do not already have one. Navigate to the [agrc organization](https://github.com/agrc/) on GitHub and find the [website source code](https://github.com/agrc/agrc.github.io). [Fork](https://help.github.com/articles/github-glossary/#fork) the [repository](https://help.github.com/articles/github-glossary/#repository) to create a personal copy of gis.utah.gov website.
+
+#### Creating and Naming New Content
+
+The gis.utah.gov website is generated using [Jekyll](http://jekyllrb.com/). Jekyll is a tool that aids in the creation of the websites content as well as the look and feel. Jekyll dictates how the website directory structure is organized.
+
+New `Posts` should be placed in the [`_posts`](https://github.com/agrc/agrc.github.io/tree/master/_posts) folder of your fork (*https://github.com/YOUR USER NAME/agrc.github.io/tree/master/_posts*). This can be accomplished by first navigating to the `_posts` folder and clicking the `New File` button. Copy and paste the content generated in the `File Path` input that the templater created.
```

Comment:
`(*https://github.com/YOUR USER NAME/agrc.github.io/tree/master/_posts*)` parses funny. Perhaps it should be something like: `(*[https://github.com/YOUR USER NAME/agrc.github.io/tree/master/_posts](https://github.com/YOUR USER NAME/agrc.github.io/tree/master/_posts)*)`?